### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oupadhyay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,30 +15,54 @@ This repository contains the frontend and backend code for CovEd's revamped webs
...
-```
+```bash
 $ npm test
 ```
+#### Linting
+If there are a lot of errors regarding the linter, run the following command to run it locally 
+```bash
+npx lerna run lint
+```
+**Caveat:** Do NOT attempt to install `eslint` as a package as it will break the code. Use the above code, but install all necessary extensions (most code editors already support it).
```
Comment (sanjayyepuri) [inline README.md:65]: Is there a particular reason someone would try to install eslint?

File: README.md [hunk 2]
```diff
@@ -15,30 +15,54 @@ This repository contains the frontend and backend code for CovEd's revamped webs
 ### 🐑 Install dependencies
 To install dependencies run the following.
 
-```
+```bash
 $ npm install
 $ npx lerna bootstrap
 ```
 
+#### Dependency Errors
```
Comment (sanjayyepuri) [inline README.md:23]: Rename this to "Adding New Dependencies."

File: README.md (hunk 1 above)
Comment (oupadhyay) [reply README.md:65]: As of now, it is not included as a package. When one attempts to run 
```
eslint
```
 an error occurs; hence, one would attempt to remedy this by installing the eslint package. However, this is a breaking change and one should just run 
```
npx lerna run lint
``` 

**Inspiration**: Me :disappointed:

File: README.md (hunk 2 above)
Comment (oupadhyay) [reply README.md]: Done

**Comment by oupadhyay** [comment]
The badges are additions that may be removed but are very cool IMO. :smiley:

**Pull request merged**